static/app/utils/events.tsx

```diff
@@ -42,7 +42,7 @@ export function getMessage(
     return event.culprit || '';
```

Comment:
I left because I have no clue what a Tombstone is but it sounds legacy and this key is legacy so I thought I'd leave it.